orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -35,12 +36,23 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
 
   @Override
   protected List<TaskNode.TaskDefinition> basicTasks(Stage stage) {
-    return [
+    def tasks = [
       new TaskNode.TaskDefinition("createServerGroup", CreateServerGroupTask),
       new TaskNode.TaskDefinition("monitorDeploy", MonitorKatoTask),
       new TaskNode.TaskDefinition("forceCacheRefresh", ServerGroupCacheForceRefreshTask),
+      ]
+
+    if (true) {
```

Comment:
For the time being, I opt'd to not add a follow up `MonitorKatoTask` was I didn't want an inadvertent failure to tag metadata to cause the deploy stage to fail.